Dockerfile.dapper

```diff
@@ -8,9 +8,7 @@ ENV https_proxy=$https_proxy
 ENV no_proxy=$no_proxy
 
 RUN apk -U --no-cache add bash git gcc musl-dev docker vim less file curl wget ca-certificates jq linux-headers zlib-dev tar zip squashfs-tools npm coreutils \
-    python2 openssl-dev libffi-dev libseccomp libseccomp-dev make libuv-static
-
-RUN apk -U --no-cache --repository http://dl-3.alpinelinux.org/alpine/edge/main/ add sqlite-dev sqlite-static libselinux libselinux-dev
+    python2 openssl-dev libffi-dev libseccomp libseccomp-dev make libuv-static sqlite-dev sqlite-static libselinux libselinux-dev zlib-dev zlib-static
```

Comment:
I feel like pinning the version of the sqlite package(s) being installed would be a good idea here. We could parameterize via build ARG.

Reply:
If we do that, should we pin everything? is sqlite more critical than all other things that we're statically linking against?